doc/api_assets/style.css

```diff
@@ -79,6 +79,15 @@ code a:hover {
   color: white !important;
 }
 
+pre.api_stability_0 a,
+pre.api_stability_1 a,
+pre.api_stability_2 a,
+pre.api_stability_3 a,
+pre.api_stability_4 a,
+pre.api_stability_5 a {
```

Comment:
It's added by the syntax-highlighting library (https://github.com/nodejs/node/blob/ccc790b6286c3a7655fb278e370eacb3ec098c4a/doc/api_assets/sh_main.js#L454) because these stability notices are intended with 4 spaces which technically makes it source code for the parser.